<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyParadox</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="pictureLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="imageLbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="pluginsLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="modsLbl">
          <property name="text">
           <string>Mod&amp;s:</string>
          </property>
          <property name="buddy">
           <cstring>modsView</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="modsView">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>100</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="dlcsLayout">
        <item>
         <widget class="QLabel" name="dlcsLbl">
          <property name="text">
           <string>&amp;DLCs:</string>
          </property>
          <property name="buddy">
           <cstring>dlcsView</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="dlcsView">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>100</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="expansionsLayout">
        <item>
         <widget class="QLabel" name="expansionsLbl">
          <property name="text">
           <string>E&amp;xpansions:</string>
          </property>
          <property name="buddy">
           <cstring>expansionsView</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="expansionsView">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>100</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="configBtn">
        <property name="text">
         <string>&amp;Configure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runBtn">
        <property name="text">
         <string>&amp;Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar"/>
 </widget>
 <tabstops>
  <tabstop>runBtn</tabstop>
  <tabstop>modsView</tabstop>
  <tabstop>dlcsView</tabstop>
  <tabstop>expansionsView</tabstop>
  <tabstop>configBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
